src/Tribe/Customizer/Global_Elements.php

```diff
@@ -121,59 +172,156 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// overrides for tec components/full/_events-bar.pcss
 			$template .= '
-				@keyframes tribe-common-c-loader-bounce {
-					0% {}
-					50% { background-color: <%= global_elements.accent_color %>; }
-					100% {}
+				.tribe-events .tribe-events-c-events-bar__search-button:before {
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
+			// overrides for tec components/full/_ical-link.pcss
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: fe3f468f5842fb57bb812f017c1a0bf2</sub>